aero.minova.core.application.system.service/doc/adoc/api-error.adoc

```diff
@@ -10,9 +10,124 @@ insert into xtcasError (Username, ErrorMessage, Date) values (username,errorMess
 ----
 
 
-Das Rückgabeformat der Tabelle, welche an den *Client* gesendet wird, ändert sich wie folgt:
+Das Rückgabeformat der Tabelle, welche an den *Client* gesendet wird, kann folgende Formate besitzen:
 
-[source,sql]
+ 1. Sql-Fehler-Format
+ 2. Programm-Fehler-Format
+
+=== Sql-Fehler-Format
+
+.Ursprünglich Fehlermeldung
+[source]
+----
+(ADO | ADO | 25 | msg.Error @p tTable.Description.7 @p tTable.Description.14 | Default Message)
+----
+
+Das Sql-Fehler Format wird dann auftreten, wenn es einen Fehler in einer oder mehreren Prozeduren gibt.
+
+Das CAS wird diesen in folgender Form ausgeben:
+
+
+.Sql-Fehler-Format
+[source,json]
+----
+{
+{
+    "resultSet": {
+        "name": "Error",
+        "columns": [
+            {
+                "name": "International Message",
+                "type": "STRING"
+            },
+            {
+                "name": "p",
+                "type": "STRING"
+            },
+            {
+                "name": "p",
+                "type": "STRING"
+            },
+            {
+                "name": "DEFAULT",
+                "type": "STRING"
+            }
+        ],
+        "rows": [
+            {
+                "values": [
+                    "s-msg.Error",
+                    "s-tTable.Description.7",
+                    "s-tTable.Description.14",
+                    "s-Default Message"
+                ]
+            }
+        ],
+	"returnErrorMessage": {
+		"detailsMessage": "ExceptionMessage",
+		"cause": "ExceptionCause",
+		"trace": [
+			"StackTraceMessages",
+			...
+		]
+	},
+    "returnCodes": [
+        -1
+    ],
+    "returnCode": -1
+}
+----
+
+
+Es wird immer im ersten Value die Kurzform der Fehlermeldung 'msg.*' stehen.
+Im letzten Value steht immer die DEFAULT Übersetzung der Fehlermeldung.
```

Comment:
In der letzten Spalte namens `DEFAULT` steht immer die Standard-Übersetzung der Fehlermeldung.